cdap-ui/app/directives/my-global-navbar/my-global-navbar.html

```diff
@@ -181,7 +181,7 @@
 <!-- SIDEBAR MENU -->
 <div class="display-container" ng-if="Navbar.showSidebar" ng-click="Navbar.toggleSidebar()">
   <div class="sidebar" ng-click="$event.stopPropagation()">
-    <a ui-sref="overview({namespace: $stateParams.namespace})"
+    <a ui-sref="overview({namespace: $stateParams.namespace || 'default' })"
```

Comment:
This could be wrong right? I could be in another namespace before going to Admin section. I add a `stream` in management screen and click on tracker. This will take it to default. Whereas when we click on cdap or hydrator it goes back to the same namespace I used to be in. 

This is a bigger issue that is right now affecting all three places (cdap, tracker & hydrator). The reason why we see everything normal in cdap and hydrator is for some reason the controller of the landing states (states that we go from admin) gets executed twice. The first time there is no namespace and the next time when the controller runs it has namespace. We have the previously visited namespace cached we need to use that consistently while navigating away from Admin section.